pkg/ebpf/tracee.go

```diff
@@ -42,15 +42,17 @@ import (
 	"github.com/aquasecurity/tracee/pkg/pcaps"
 	"github.com/aquasecurity/tracee/pkg/policy"
 	"github.com/aquasecurity/tracee/pkg/signatures/engine"
+	"github.com/aquasecurity/tracee/pkg/streams"
 	"github.com/aquasecurity/tracee/pkg/utils"
 	"github.com/aquasecurity/tracee/pkg/utils/proc"
 	"github.com/aquasecurity/tracee/pkg/utils/sharedobjs"
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
 const (
-	pkgName          = "tracee"
-	maxMemDumpLength = 127
+	pkgName            = "tracee"
+	maxMemDumpLength   = 127
+	allPoliciesEnabled = 0xffffffffffffffff
```

Comment:
I had this in my PR:

![image](https://github.com/aquasecurity/tracee/assets/7395852/caa3ac03-6c25-45cb-ae55-3627a60f5bf6)

and then I also replaced:

![image](https://github.com/aquasecurity/tracee/assets/7395852/6a9a5664-f915-472d-898c-c9a3d002fc50)

What do you think ?

Reply:
Nice! Is this in a commit I can cherry-pick? Or do you mind if I do the change here?

Reply:
Not at all, put this change in your PR, I'll rebase once yours is merged.